docs/profile/verifications.rst

```diff
@@ -207,6 +207,43 @@ Example
 
 
 
+--------------------------------------------------------------------------------
+
+.. _verifications_getOwnerAddressForIdentity:
+
+getOwnerAddressForIdentity
+================================================================================
+
+.. code-block:: typescript
+
+  verifications.getOwnerAddressForIdentity(identityAddress);
+
+Gets the identity contract for a given account id or contract.
```

Comment:
Should we update description due to logic changes?
```suggestion
Gets an identity's owner's address. This can be either an account or an identity address.
```